packages/blockchain-ui/src/theme/colors.ts

```diff
@@ -65,4 +77,26 @@ export const colors = {
   error900: "#782424",
 };
 
-
+/**
+ * All the theme colors declarations here..
+ */
+export const themeColors: Record<TColorSchemes, TSchemeColors> = {
+  dark: {
+    text: {
+      primary: "#FFFFFF",
+      disabled: "#637381",
+    },
+    background: {
+      main: "#212B36",
+    },
+  },
+  light: {
+    text: {
+      primary: "#212B36",
```

Comment:
The light and dark theme objects are based on different.
If you need to create dark theme just createTheme function and override the fields that you want to override